scripts/future-fetch.py

```diff
@@ -45,6 +45,9 @@ class Entry(object):
     witnesses_str = ''
     location = ''
 
+    # strip 4 byte unicode chars plus zero width space
+    re_pattern = re.compile(u'(?:[^\u0000-\uD7FF\uE000-\uFFFF]|[\u200B])', re.UNICODE)
```

Comment:
I think that converting the table to UTF-8 is going to be the easiest solution here as the character conversion option generates further problems when the conversion for the DB connection happens.